### DISCUSSION TITLE
How to embed a Livewire component into schema? (v3)

### DISCUSSION THREAD

**Discussion by njxqlus** [filamentphp/filament#7735] [Help]
Previously I asked [how to insert custom HTML between two fields in form](https://github.com/filamentphp/filament/discussions/7659) and still not found a better answer than inserting it like `\Filament\Forms\Components\View::make()`. It works for simple HTML but now I need to embed a Livewire component into form, table, etc.

One but not the only point of it is to show, for example, a Filament table in an Infolist.

I achieved with next steps:
1. `php artisan make:infolist-layout Bar`
2. `php artisan make:livewire Foo`
3. Add `<div>@livewire('foo')</div>` to bar.blade.php
4. Add `\Filament\Forms\Components\View::make('infolists.components.bar')` to the Resource
5. Add table suggesting [instruction](https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component) to Foo Livewire component

It works but `bar.blade.php` looks dumb. My question is am I doing these things right? Is there an option like `Livewire::make()` maybe?

**Comment by danharrin** [comment]
You can PR a new `Livewire` component class if you want

**Comment by njxqlus** [comment]
PR #7743
